cpp/include/cudf/column/column_factories.hpp

```diff
@@ -229,6 +229,37 @@ std::unique_ptr<column> make_strings_column(
     cudaStream_t stream = 0,
     rmm::mr::device_memory_resource* mr = rmm::mr::get_default_resource());
 
+/**
+ * @brief Construct STRING type column given a device vector of string_view.
+ * The total number of char bytes must not exceed the maximum size of size_type.
+ * The string characters are expected to be UTF-8 encoded sequence of char
+ * bytes. Use the strings_column_view class to perform strings operations on
+ * this type of column.
+ *
+ * @note `null_count()` and `null_bitmask` are determined if a string_view is
```

Comment:
I think rewrite this @note as

```
@note For each string_view, if `.data()` is `null_placeholder.data()`, that string is considered null. Likewise, a string is considered empty (not null) if `.data()` is not `null_placeholder.data()` and `.size_bytes()` is 0. Otherwise the `.data()` must be a valid device address pointing to `.size_bytes()` consecutive bytes. The `null_count()` for the output column will be equal to the number of input `string_view`s that are null.
```